include/dem/particle_wall_broad_search.h

```diff
@@ -141,19 +141,19 @@ class ParticleWallBroadSearch
 
   void
   particle_floating_mesh_contact_search(
```

Comment:
The name of the function is still `particle_floating_mesh_contact_search()`

Reply:
good catch